src/js/components/Nav/Nav.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import { Anchor } from '../Anchor';
+import { Box } from '../Box';
+
+
+ // TODO resolve whether children should be allowed with items
```

Comment:
Opening this for discussion. I think there shouldn't be a problem to allow both.